### ISSUE TITLE
Read from different yaml file

### ISSUE THREAD

**Issue by n4cr** [vifreefly/procsd#4]
Is it possible to use a different yml file? I have multiple configurations for different servers and I would like to use a different file for each server. At the moment I use a different Procfile for each.

**Comment by vifreefly** [comment]
No, at the moment it's not possible to provide different filename other than `procsd.yml`. But it's a good idea, will add it in the next version :+1:
